src/ngraph/runtime/gpu/cudnn_emitter.hpp

```diff
@@ -50,6 +50,7 @@ namespace ngraph
                 std::vector<int> get_vector_int_from_size_t(const std::vector<size_t>&);
             }
             class GPUPrimitiveEmitter;
+            class GPUAllocator;
```

Comment:
nit: Is this needed?

Reply:
Thnks :) Good catch! This was from a previous version of my code where I had wrapper function for getting the workspace.